docs/tutorial/code-signing.md

```diff
@@ -1,8 +1,15 @@
 # Code Signing
 
 Code signing is a security technology that you use to certify that an app was 
-created by you. Once an app is signed, the system can detect any change to the 
-app, whether the change is introduced accidentally or by malicious code.
+created by you.
+
+On macOS the system can detect any change to the  app, whether the change is
+introduced accidentally or by malicious code.
+
+On Windows the system assigns a trust level to your code signing certificate which
+if you don't have, or if your trust level is low will cause security dialogs to
```

Comment:
Suggested rewording:

> On Windows the system assigns a trust level to your code signing certificate. If you haven't yet established a trust level or your trust level is low, security dialogs will appear when users start using your application. Your trust level builds over time, so it's better to start code signing as early as possible.

Reply:
☝️ Is there a good page we can link to that explains this in more detail?

Reply:
https://blogs.msdn.microsoft.com/ieinternals/2011/03/22/everything-you-need-to-know-about-authenticode-code-signing/

Looks like Microsoft refer to it as "Reputation" not "trust level"